### PR TITLE
Make some minor language and formatting changes to the PR template

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request modifies the language and formatting of the PR template to:
* Fix some line wrapping in the Markdown source that was rendering as actual line breaks
* Tweak the language around the test coverage bullet to allow for modifying existing tests

## 💭 Motivation and Context ##

Beauty and clarity are paramount!

## 🧪 Testing ##

All pre-commit hooks pass.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
LGTM 🐐

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -35,11 +35,9 @@
...
-* [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md)
-      document.
-* [ ] These code changes follow [cisagov code
-      standards](https://github.com/cisagov/development-guide).
+* [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
+* [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
 * [ ] All relevant repo and/or project documentation has been updated
       to reflect the changes in this PR.
-* [ ] Tests have been added to cover the changes in this PR.
+* [ ] Tests have been added and/or modified to cover the changes in this PR.
```
Comment (mcdonnnj) [inline .github/pull_request_template.md:42]: Would this be a simpler catch-all to use?
```suggestion
* [ ] Tests have been updated to cover the changes in this PR.
```

**Review by mcdonnnj — APPROVED** [review]
LGTM ✅ Thanks for continuing to refine this! I had one take-it-or-leave-it comment for your consideration.

File: .github/pull_request_template.md (hunk 1 above)
Comment (hillaryj) [reply .github/pull_request_template.md:42]: I think it's worth calling out "added" to put it in peoples' heads in case that hadn't occurred to them.

**Review by hillaryj — APPROVED** [review]
🦖

File: .github/pull_request_template.md (hunk 1 above)
Comment (jsf9k) [reply .github/pull_request_template.md:42]: I like the slightly more explicit language in the original.

Comment (jsf9k) [reply .github/pull_request_template.md:42]: image

**Pull request merged**